internal/util/crypto.go

```diff
@@ -197,19 +197,20 @@ func VolumeMapper(volumeID string) (mapperFile, mapperFilePath string) {
 // EncryptVolume encrypts provided device with LUKS.
 func EncryptVolume(ctx context.Context, devicePath, passphrase string) error {
 	log.DebugLog(ctx, "Encrypting device %s with LUKS", devicePath)
-	if _, _, err := LuksFormat(devicePath, passphrase); err != nil {
-		return fmt.Errorf("failed to encrypt device %s with LUKS: %w", devicePath, err)
+	_, stdErr, err := LuksFormat(devicePath, passphrase)
+	if err != nil || stdErr != "" {
+		log.ErrorLog(ctx, "failed to encrypt device %s with LUKS: error:%v stdError:%s", devicePath, err, stdErr)
```

Comment:
nit: the logging gets quite ugly this way, maybe do

    failed to encrypt device %q with LUKS (%v): %s

Reply:
Done. logging with same format.